recipe/meta.yaml

```
@@ -26,15 +26,15 @@ requirements:
     - anyio >=4.5
     - httpx >=0.27
     - httpx-sse >=0.4
-    - pydantic >=2.7.2,<3.0.0
+    - pydantic >=2.8.0,<3.0.0
     - pydantic-settings >=2.5.2
-    # - python-dotenv >=1.0.0
     - python-multipart >=0.0.9
     - rich >=13.9.4
     - sse-starlette >=1.6.1
     - starlette >=0.27
-    # - typer >=0.12.4
-    - uvicorn >=0.23.1
+    - uvicorn >=0.23.1    # [not emscripten]
+    - jsonschema >=4.20.0
+    - pywin32 >=310  # [win]
```

There are some potential `run_constrained` from https://github.com/modelcontextprotocol/python-sdk/blob/v1.12.1/pyproject.toml#L38-L41

```suggestion
    - pywin32 >=310  # [win]
  run_constrained:
    - typer >=0.16.0
    - python-dotenv >=1.0.0
    - rich >=13.9.4
    - websockets >=15.0.1
```

(identified by `recipe-bumper-cli` -- I'm just here to take the credit...)

---

I'll add them :)